lib/gcp/runtimeconfig.js

```diff
@@ -8,10 +8,28 @@ var _ = require('lodash');
 
 var API_VERSION = 'v1beta1';
 
+function _retryOnServerError(requestFunction) {
+  var retryableCodes = [500, 503];
```

Comment:
For constants that are only used in one place, it's usually more clear just to inline them (recent advice from Go I got that I think applies here).

Let's put both retryableCodes and waitTime inline.